nuxeo-services/nuxeo-platform-convert/src/main/java/org/nuxeo/ecm/platform/convert/plugins/LibreOfficeConverter.java

```diff
@@ -79,14 +81,14 @@ protected BlobHolder buildResult(List<String> cmdOutput, CmdParameters cmdParams
     }
 
     private void deleteTempDirectory(String tempPath) {
-        Path path = Paths.get(tempPath);
-        if (!Files.exists(path)) {
-            return;
-        }
-
         try {
-            FileUtils.deleteDirectory(path.toFile());
-        } catch (IOException e) {
+            URI uri = new URI(tempPath);
+            File file = new File(uri);
+            if (!Files.exists(file.toPath())) {
```

Comment:
Worth a comment explaining why we go through a URI.
Also maybe `Paths.get(uri)` could be used? Although you still need a `File` for `FileUtils.deleteDirectory` so maybe not worth it...

Reply:
I've just backported the existing code on master from fa4958249df128f92f61d3c0e13464e93b14dc6f - so I would prefer to not update it only on backports.
Still, do you think the master code should be updated to use `Paths.get(uri)`?

Reply:
No, it's a detail.

Reply:
I'll just add a comment to explain that we get here an URI such as `file://xxx` on all branches.